.github/workflows/kind.yml

```
@@ -250,7 +250,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v5
         with:
-          go-version-file: go.mod
+          go-version: '1.24'
```

Actually, looks like there's a tag we can use instead. It'll save us from thinking about it again.

ref: https://github.com/actions/setup-go?tab=readme-ov-file#using-stableoldstable-aliases


```suggestion
          go-version: 'stable'
```

---

Great Thanks!!